tests/build-arg-dynamic-with-empty-base.earth

```diff
@@ -0,0 +1,10 @@
+# This test requires that the base be empty (i.e. no FROM)
```

Comment:
What's the significance of the base being empty here? Does it impact how the ARG is evaluated?

Reply:
This was part of the repo-case for this bug. With this fix this becomes irrelevant.